### PR TITLE
Added Django sample for App Engine Python 3.7 standard runtime 

### PR DESCRIPTION
**Pull request by michaelawyu**

### CODE REVIEW

File: appengine/standard_python37/django/.gcloudignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+.gcloudignore
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
+.git
+.gitignore
+
+# Python pycache:
+__pycache__/
```
Comment (andrewsg) [inline appengine/standard_python37/django/.gcloudignore]: nit: add newline

File: appengine/standard_python37/django/app.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# [START django_app]
+runtime: python37
+entrypoint: gunicorn -b :$PORT mysite.wsgi
```
Comment (andrewsg) [inline appengine/standard_python37/django/app.yaml]: Is there any way we can do this without specifying a manual entrypoint, instead using the default entrypoint?

If there is not (I know Django is more challenging in this respect) then please change this entrypoint line to add multiple workers and threads. We should make every effort to use the default without an entrypoint though, even if it is awkward.

File: appengine/standard_python37/django/app.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# directory.
+- url: /static
+  static_dir: static_files/
+
+# This handler routes all requests not caught above to your main app. It is
+# required when static routes are defined, but can be omitted (along with
+# the entire handlers section) when there are no static files defined.
+- url: /.*
+  script: auto
+# [END django_app]
```
Comment (andrewsg) [inline appengine/standard_python37/django/app.yaml]: nit: add newline

File: appengine/standard_python37/django/mysite/settings.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'lldtg$9(wi49j_hpv8nnqlh!cj7kmbwq0$rj7vy(b(b30vlyzj'
```
Comment (andrewsg) [inline appengine/standard_python37/django/mysite/settings.py]: Do our other Django samples hardcode a secret key like this? It seems like we should have a replace-me placeholder or something, but you can follow the lead of our other Django samples if needed.

File: appengine/standard_python37/django/mysite/settings.py [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+# app not on App Engine, make sure to set an appropriate host here.
+# See https://docs.djangoproject.com/en/2.1/ref/settings/
+ALLOWED_HOSTS = ['*']
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'polls.apps.PollsConfig',
+    'django.contrib.admin',
```
Comment (andrewsg) [inline appengine/standard_python37/django/mysite/settings.py]: Are all these contrib apps used? I'm not sure whether or not we should include them if they are unused. We can follow the lead of our other Django samples.

File: appengine/standard_python37/django/polls/admin.py [hunk 6]
```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+from .models import Question, Choice
+
+admin.site.register(Question)
+admin.site.register(Choice)
```
Comment (andrewsg) [inline appengine/standard_python37/django/polls/admin.py]: nit: add newline

File: appengine/standard_python37/django/polls/templates/polls/index.html [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{% if latest_question_list %}
+    <ul>
+    {% for question in latest_question_list %}
+        <li><a href="{% url 'polls:detail' question.id %}">{{ question.question_text }}</a></li>
+    {% endfor %}
+    </ul>
+{% else %}
+    <p>No polls are available.</p>
+{% endif %}
```
Comment (andrewsg) [inline appengine/standard_python37/django/polls/templates/polls/index.html]: nit: add newline

File: appengine/standard_python37/django/polls/views.py [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+            'question': question,
+            'error_message': "You didn't select a choice.",
+        })
+    else:
+        selected_choice.votes += 1
+        selected_choice.save()
+        # Always return an HttpResponseRedirect after successfully dealing
+        # with POST data. This prevents data from being posted twice if a
+        # user hits the Back button.
+        return HttpResponseRedirect(reverse('polls:results', args=(question.id,)))
```
Comment (andrewsg) [inline appengine/standard_python37/django/polls/views.py]: nit: add newline

File: appengine/standard_python37/django/requirements.txt [hunk 9]
```diff
@@ -0,0 +1,2 @@
+Django==2.1.1
+mysqlclient==1.3.13
```
Comment (andrewsg) [inline appengine/standard_python37/django/requirements.txt]: Can we use pymysql here instead? We're trying to standardize on that across our samples as complaints about installing mysql dev libs are very common.

File: appengine/standard_python37/django/mysite/settings.py (hunk 5 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/mysite/settings.py]: Andrew, in this tutorial I just used the default set of contrib apps listed in the Django getting started tutorial (https://docs.djangoproject.com/en/2.1/intro/tutorial02/); I can remove the unused ones if required though.

File: appengine/standard_python37/django/.gcloudignore (hunk 1 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/.gcloudignore]: Done.

File: appengine/standard_python37/django/app.yaml (hunk 3 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/app.yaml]: Done.

File: appengine/standard_python37/django/polls/admin.py (hunk 6 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/polls/admin.py]: Done.

File: appengine/standard_python37/django/polls/templates/polls/index.html (hunk 7 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/polls/templates/polls/index.html]: Done.

File: appengine/standard_python37/django/polls/views.py (hunk 8 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/polls/views.py]: Done.

File: appengine/standard_python37/django/requirements.txt (hunk 9 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/requirements.txt]: Got it. Done.

File: appengine/standard_python37/django/mysite/settings.py (hunk 4 above)
Comment (michaelawyu) [reply appengine/standard_python37/django/mysite/settings.py]: Yeah, our App Engine Python 2.7 Django sample uses the sample pattern as well. I will add a comment to remind developers of changing the secret key.

Comment (michaelawyu) [reply appengine/standard_python37/django/mysite/settings.py]: Added. Weirdly the snapshot won't refresh :(

**Pull request merged**